src/mplfinance/plotting.py

```diff
@@ -787,22 +775,29 @@ def _addplot_columns(panid,panels,ydata,apdict,xdates,config):
         mark  = apdict['marker']
         color = apdict['color']
         alpha = apdict['alpha']
+        labels = apdict["labels"]
         if isinstance(mark,(list,tuple,np.ndarray)):
-            _mscatter(xdates,ydata,ax=ax,m=mark,s=size,color=color,alpha=alpha)
+            _mscatter(xdates,ydata,ax=ax,m=mark,s=size,color=color,alpha=alpha) #labels in this function needs to be added
         else:
             ax.scatter(xdates,ydata,s=size,marker=mark,color=color,alpha=alpha)
+            if apdict["labels"] is not None:
```

Comment:
Since `apdict["labels"]` was assigned to `labels` above, then `labels` should be used here and below.